### PR TITLE
issue #54 - Replace 'request()->' with 'Request::' to Laravel 5.0 com…

### PR DESCRIPTION
**Pull request by CharlesHenry**
Fix Laravel 5.0 compatibility (issue #54)

### CODE REVIEW

**Comment by rap2hpoutre** [comment]
Ok thanks! Does this work from laravel 4.2 to laravel 5.3?

File: src/controllers/LogViewerController.php [hunk 1]
```diff
@@ -14,14 +14,14 @@ class LogViewerController extends Controller
 
     public function index()
     {
-        if (request()->input('l')) {
-            LaravelLogViewer::setFile(base64_decode(request()->input('l')));
+        if (Request::input('l')) {
```
Comment (mikemand) [inline src/controllers/LogViewerController.php]: This should be `Request::has('l')`.

File: src/controllers/LogViewerController.php [hunk 2]
```diff
@@ -14,14 +14,14 @@ class LogViewerController extends Controller
...
+            LaravelLogViewer::setFile(base64_decode(Request::input('l')));
         }
 
-        if (request()->input('dl')) {
-            return Response::download(storage_path() . '/logs/' . base64_decode(request()->input('dl')));
-        } elseif (request()->has('del')) {
-            File::delete(storage_path() . '/logs/' . base64_decode(request()->input('del')));
+        if (Request::input('dl')) {
```
Comment (mikemand) [inline src/controllers/LogViewerController.php]: Same for here. `Request::has('dl')`

**Comment by CharlesHenry** [comment]
@rap2hpoutre, yes it's using the Request class.

@mikemand I agree with you but I think that's a different change to what I'm doing here. That issue you are highlighting already existed before I made my change and it's functional how it is.

**Comment by rap2hpoutre** [comment]
Ok thank you! Available in v0.5.3

**Pull request merged**